packages/lesswrong/components/Layout.tsx

```diff
@@ -70,6 +70,9 @@ const styles = (theme: ThemeType): JssStyles => ({
     // but almost all pages are bigger than this anyway so it's not that important
     minHeight: `calc(100vh - ${HEADER_HEIGHT}px)`,
     gridArea: 'main',
+    [theme.breakpoints.down('md')]: {
+      paddingTop: isFriendlyUI ? 0 : theme.spacing.mainLayoutPaddingTop,
+    },
```

Comment:
I noticed we have a weird middle range where there is extra padding right under the header. This is not the case for LW. So I adjusted the padding here slightly.